src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -230,13 +229,17 @@ def get_login_credentials(self, credential_type=CredentialType.management,
         account = self.get_subscription(subscription_id)
         user_type = account[_USER_ENTITY][_USER_TYPE]
         username_or_sp_id = account[_USER_ENTITY][_USER_NAME]
+        try:
+            resource = credential_type.value
```

Comment:
Yes, I retrieve the resource from the ENDPOINT_URLS. Turns out I had that in a separate branch but I merged it into this one. So it looks like:
`profile.get_login_credentials(resource=get_env()[ENDPOINT_URLS.ACTIVE_DIRECTORY_GRAPH_RESOURCE_ID])`